.gcloudignore

```diff
@@ -4,4 +4,4 @@
 .git
 .gitignore
 #!include:.gitignore
-vendor
+#vendor
```

Comment:
I am not sure why we remove the ignore

Reply:
Is it because since this PR focus on building completely on gcloud using docker file, vendor directory is needed to be uploaded to gcloud?

Reply:
@hmtai